### ISSUE TITLE
[NPC] Burning Depths Necromancer does not attack at range.

### ISSUE THREAD

**Issue by Wackmore** [dalaranwow/dalaran-wow#5284]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: I ran into a problem while questing tru Maw of Naltharion (dragonblight).
The mob ¨Burning Depths Necromancer¨ (npc=27358) does not attack at range, nor does the mob run to you while having agro. This was on the alagon realm.

I hope this helps :) 
   -wackmore

**Expected behaviour**: The mob should attack at range, or run to the character for melee combat.

**Steps to reproduce the problem**:

1. use a ranged class.
2. attack ¨Burning Depths Necromancer¨ (npc=27358)  with a ranged attack.
3. -

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

![screenshot_20180204_191218](https://user-images.githubusercontent.com/36135461/35780682-7d32e38a-09df-11e8-8918-2c53f6a3f702.png)

**Include Screenshots from the issue if necessary**
